### ISSUE TITLE
lint: methodExprCall for nil receiver

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#681]
From goroot:
```
$GOROOT/src/runtime/stack_test.go:649:37: methodExprCall: consider to change `I.M` to `nil.M`
```
This is a false positive.
